pkg/ebpf/tracee.go

```diff
@@ -275,7 +275,16 @@ func New(cfg config.Config) (*Tracee, error) {
 
 	// Initialize capabilities rings soon
 
-	err = capabilities.Initialize(t.config.Capabilities.BypassCaps)
+	useBaseEbpf := func(cfg config.Config) bool {
+		return cfg.Output.StackAddresses
+	}
```

Comment:
Can't we simply add the required bpf capability to the base ring if StackAddresses is selected?

Reply:
So if in the future we add other features which require a similar approach, we can add the logic here instead of directly coupling to stackaddresses.

Reply:
Why coupling?
I mean, to expose some method to add more capabilities to the base ring, and then we can remove all the other changes in this PR, and just add the capability using this method

Reply:
There already is such a method, but the eBPF caps specifically aren't constant across systems. So I thought initializing a specific bypass for eBPF in the code location where we do the calculation of what are these eBPF caps, would be easier.
About coupling, I meant that this is slightly less coupled, or at least that it makes the intention that this condition coud change in the future more clear, then doing `ebpfBase: cfg.Output.StackAddresses`.